.circleci/config.yml

```diff
@@ -14,6 +14,10 @@ install_and_test: &install_and_test
         name: Test
         command: npm test
         when: always
+    - run:
+        name: System Tests
+        command: npm run system-tests
+        when: always
```

Comment:
Maybe this shouldn’t be always, so that if decrypt fails, this step will be skipped.